### ISSUE TITLE
error raised if a feature and a (renamed) package have the same name

### ISSUE THREAD

**Issue by lyro** [kbknapp/cargo-outdated#207]
If you have a feature with the same name, say `xyz`, and a package with the same name then an error is raised:

```
error: failed to parse manifest at `/tmp/cargo-outdatedVUSS6n/Cargo.toml`

Caused by:
  Features and dependencies cannot have the same name: `crossbeam`
```

This happens even if the package has been renamed:

```
[features]
xyz = ["pkg-xyz"]

[dependencies]
pkg-xyz = { version = "*", optional = true, package = "xyz" }
```
All regular cargo commands allow this.

**Comment by deg4uss3r** [comment]
Hi @lyro Thanks for this issue. 

I found out the upstream issue is actually https://github.com/alexcrichton/toml-rs/issues/265 

I have a really ugly patch that gets this working in https://github.com/deg4uss3r/cargo-outdated/tree/namespace 

I hope to clean that up later this week and have a work in progress PR going.

**Comment by deg4uss3r** [comment]
Fixed by #208
